apiserver/pkg/server/cluster_server.go

```diff
@@ -97,7 +97,8 @@ func (s *ClusterServer) ListCluster(ctx context.Context, request *api.ListCluste
 // Finds all Clusters in all namespaces.
 // TODO: Supports pagination and sorting on certain fields when we have DB support. request needs to be extended.
 func (s *ClusterServer) ListAllClusters(ctx context.Context, request *api.ListAllClustersRequest) (*api.ListAllClustersResponse, error) {
-	clusters, err := s.resourceManager.ListAllClusters(ctx)
+	// Leave the namespace empty to list all clusters in all namespaces.
+	clusters, continueToken, err := s.resourceManager.ListClusters(ctx, /*namespace=*/ "", request.Continue, request.Limit)
```

Comment:
post-merge comment, I think have issues with our linter; if you do `make lint` under `apiserver` folder explicitly, it will see formatting getting fixed.
```sh
        // Leave the namespace empty to list all clusters in all namespaces.
-       clusters, continueToken, err := s.resourceManager.ListClusters(ctx, /*namespace=*/ "", request.Continue, request.Limit)
+       clusters, continueToken, err := s.resourceManager.ListClusters(ctx /*namespace=*/, "", request.Continue, request.Limit)
```